### DISCUSSION TITLE
Start Z2M with an invalid adapter

### DISCUSSION THREAD

**Discussion by spudwebb** [Koenkk/zigbee2mqtt#17708] [General]
According to my tests if the Zigbee adapter is not correctly configured in configuration.yaml, Z2M fails to start.
Is there a way to let Z2M starts even if the adapter is not configured, so that the configuration can be done via MQTT or the front-end rather than manually editing  configuration.yaml?

**Comment by lux73** [comment]
AFAIK not possible
